rust/Cargo.lock

```diff
@@ -24,6 +24,15 @@ dependencies = [
  "arrow-select",
 ]
 
+[[package]]
+name = "adbc_snowflake"
+version = "0.15.0"
```

Comment:
probably need to update the release scripts somehow to make sure that we automatically update this upon releases and such.

Reply:
The lock file is updated automatically by cargo. The package uses the workspace version, which is already updated by the version bump script. I added this crate to publish script in https://github.com/apache/arrow-adbc/pull/2207/commits/041f72ded9b594efa950454a6284385319ef3f9e.